npdarrington/instaquizzes
# Nd styling

### What’s this PR do?  
- Added initial styling for the App.
- Added 2 new font styles that are used within the page.
- Added background image for App.
- Installed styled-components and @types/styled-components libraries.
 
### Where should the reviewer start?  
- Git pull this branch.
- `npm install` to install new dependencies.
 
### How should this be manually tested?  
- `npm start` -> localhost: 3000.
- Ensure that BG image shows and new styling for InstaQuizzes title and font in buttons.
 
### Any background context you want to provide?  
- N/A
 
### What are the relevant tickets?  
- closes #73
